x-pack/plugins/ml/public/formatters/abbreviate_whole_number.js

```diff
@@ -17,6 +17,6 @@ export function abbreviateWholeNumber(value, maxDigits) {
   if (Math.abs(value) < Math.pow(10, maxNumDigits)) {
     return value;
   } else {
-    return numeral(value).format('0a');
```

Comment:
The use of `0a` needs to be kept for the existing use of this formatter in the Top Influencers list on the Anomaly Explorer. That format was chosen so it works with the size given to the `div` for the total score label:

![image](https://user-images.githubusercontent.com/7405507/58246783-e33fac00-7d4f-11e9-8898-43945ee0f38b.png)

Might need an extra `abbreviate_number.js` type file, or else the addition of a format argument.